### ISSUE TITLE
Change `padded-blocks` to allow either, but enforce symmetry

### ISSUE THREAD

**Issue by lazarljubenovic** [eslint/eslint#14813]
**What rule do you want to change?**

`padded-blocks`

**Does this change cause the rule to produce more or fewer warnings?**

Fewer.

**How will the change be implemented? (New option, new default behavior, etc.)?**

A new option.

**Please provide some example code that this change will affect:**

```js
class Foo {
  constructor () { }

}

class Bar {

  constructor () { }

}

class Baz {
  constructor () { }
}
```

**What does the rule currently do for this code?**

Either always add padding lines, or always remove them.

```js
class Foo {

  constructor () { }

}

class Bar {

  constructor () { }

}

class Baz {

  constructor () { }

}
```

```js
class Foo {
  constructor () { }
}

class Bar {
  constructor () { }
}

class Baz {
  constructor () { }
}
```

**What will the rule do after it's changed?**

It will report asymmetrical padding lines, but will allow both "no padding lines" and "one padding line" (as long as it's the same on both sides).

```js
// eslint error/warn: padding lines not symmetrical
class Foo {
  constructor () { }

}

class Bar {

  constructor () { }

}

class Baz {
  constructor () { }
}
```

**Are you willing to submit a pull request to implement this change?**

I'm not familiar with eslint codebase.

**Rationale**

I want to leave it to the developer's discretion to decide when padding lines should or should not be added, as long as the developer follows the same convention at the top and at the bottom.

**Comment by nzakas** [comment]
Per our [README](https://github.com/eslint/eslint#stylistic-rule-updates), we are no longer making changes to rules that enforce stylistic preferences. You can always create a copy of the existing rule and modify it to do what you want, using it as a custom rule.

**Comment by lazarljubenovic** [comment]
Might wanna leave that in the issue template instead at the bottom of a README 😄
